modules/modelLoader/stableDiffusion3/StableDiffusion3EmbeddingLoader.py

```diff
@@ -66,41 +67,43 @@ def __load_internal(
 
             if os.path.exists(safetensors_embedding_name):
                 return self.__load_embedding(safetensors_embedding_name)
-            else:
-                return self.__load_embedding(embedding_name.model_name)
-        else:
-            raise Exception("not an internal model")
+
+            return self.__load_embedding(embedding_name.model_name)
```

Comment:
Another case of a removed else branch. Technically not needed, but I still prefer to keep it to make the code easier to read.